package.json

```diff
@@ -81,6 +81,7 @@
     "babel-eslint": "10.0.1",
     "babel-jest": "24.8.0",
     "babel-loader": "8.0.6",
+    "babel-plugin-styled-components": "^1.10.0",
```

Comment:
I think finally we decide to not include the styled-components, actually looks like you are not using them. Anyway, we always use exact version in our package.json :)

Reply:
Good to note about package version 😄